test/runtests.jl

```diff
@@ -19,30 +19,59 @@ img_eps = isinteractive() ? 1e-2 : 10e-2
 #     image_comparison_facts(:gadfly, skip=[4,6,23,24,27], eps=img_eps)
 # end
 
+facts("GR") do
+    @fact gr() --> Plots.GRBackend()
+    @fact backend() --> Plots.GRBackend()
+
+    image_comparison_facts(:gr, eps=img_eps)
+end
+
 facts("PyPlot") do
     @fact pyplot() --> Plots.PyPlotBackend()
     @fact backend() --> Plots.PyPlotBackend()
 
-    image_comparison_facts(:pyplot, skip=[6,25,30,31], eps=img_eps)
+    image_comparison_facts(:pyplot, skip=[2,31], eps=img_eps)
 end
 
-facts("GR") do
-    @fact gr() --> Plots.GRBackend()
-    @fact backend() --> Plots.GRBackend()
-
-    if is_linux() && isinteractive()
-        image_comparison_facts(:gr, skip=[2,25,30,31], eps=img_eps)
+facts("PlotlyJS") do
+    @fact plotlyjs() --> Plots.PlotlyJSBackend()
+    @fact backend() --> Plots.PlotlyJSBackend()
+
+    if is_linux()
+        image_comparison_facts(:plotlyjs,
+            skip=[
+                2,  # animation
+                10, # match_dimensions not defined
```

Comment:
match_dimensions is just a warning, right? It still makes sense to skip it (we don't want warnings in the tests when they run right) but I am thinking whether we should adress this in the code - there is nothing wrong with just adding it to the supported keywords for the non-pyplot backends, I think (at least those, like GR and Pyplot, that call `transpose_z` in the backend code).